.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,12 +13,15 @@ to ensure the following steps have been taken:
 
 ### For code changes:
 - [ ] Have you written and/or updated unit tests to verify your changes?
-- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
+- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
 - [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
 - [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?
 
 ### For documentation related changes:
 - [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
+- [ ] If this PR is a documentation-only change, have you added a `[skip ci]`
+  tag to the first line of your commit and your PR title to avoid spending CPU
+  cycles when no code or tests are modified?
```

Comment:
* "have you added `[skip ci]` to the first line of your commit message"
* I thought it didn't need to be in the PR title if it's in the first line of the commit message. It would only needs to be in the PR title if it isn't in the first line of the commit. The merge commit message will include the first line of source commit message.
* "avoid spending CPU cycles **in Travis CI** when..."
* "when no code, tests **or build configuration** are modified"

Reply:
@sjudeng – all changes made as suggested.